paket.lock

```diff
@@ -1,10 +1,10 @@
 CONTENT: NONE
 NUGET
   remote: https://api.nuget.org/v3/index.json
-    Argu (3.7)
-      FSharp.Core (>= 4.0.1.7-alpha) - restriction: >= netstandard1.6
-      NETStandard.Library (>= 1.6) - restriction: >= netstandard1.6
-      System.Xml.XDocument (>= 4.0.11) - restriction: >= netstandard1.6
+    Argu (5.1)
+      FSharp.Core (>= 4.0.0.1) - restriction: >= net45
+      FSharp.Core (>= 4.3.2) - restriction: && (< net45) (>= netstandard2.0)
```

Comment:
@forki I think we introduced some regression again in Paket, two times `FSharp.Core` is not good